components/fieldlabel/metadata/mods.md

```diff
@@ -1,15 +1,16 @@
-| Modifiable Custom Properties          |
-| ------------------------------------- |
-| `--mod-disabled-content-color`        |
-| `--mod-field-label-text-to-asterisk`  |
-| `--mod-field-label-top-to-asterisk`   |
-| `--mod-fieldlabel-font-size`          |
-| `--mod-fieldlabel-font-weight`        |
-| `--mod-fieldlabel-line-height`        |
-| `--mod-fieldlabel-line-height-cjk`    |
-| `--mod-fieldlabel-min-height`         |
-| `--mod-fieldlabel-side-padding-right` |
-| `--mod-fieldlabel-side-padding-top`   |
-| `--mod-spacing-300`                   |
-| `--mod-tableform-border-spacing`      |
-| `--mod-tableform-margin`              |
+| Modifiable Custom Properties                |
+| ------------------------------------------- |
+| `--mod-disabled-content-color`              |
+| `--mod-field-label-asterisk-vertical-align` |
+| `--mod-field-label-text-to-asterisk`        |
+| `--mod-field-label-top-to-asterisk`         |
+| `--mod-fieldlabel-font-size`                |
```

Comment:
I hadn't noticed it before, but our naming is inconsistent here, and I'm on the fence as to whether we should leave it alone or update it so that things are consistent. I think we might just want to leave it alone.